presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/WindowMatcher.java

```diff
@@ -46,24 +46,23 @@ public boolean matches(PlanNode node, Session session, Metadata metadata, Symbol
         }
 
         WindowNode windowNode = (WindowNode) node;
-        Collection<FunctionCall> actualCalls = windowNode.getWindowFunctions().values();
+        List<FunctionCall> actualCalls = windowNode.getWindowFunctions().values().stream().map(function -> function.getFunctionCall()).collect(toList());
```

Comment:
Done in 4131e9795952debf7b3ab5c8a7e17980955afbdf.
Though, changes from @ebd2's commit (f63b4bef45685d51abe9e49339907d1e26061960) interfere. Will fix that when you finish reviewing changes commit-by-commit.